styles.css

```diff
@@ -89,4 +100,318 @@
     top: 8px;
     border-radius: 0;
   }
+
+  /* My recent works section */
+
+  .recent-works {
+    background-color: #e5e5e5;
+    padding-left: 24px;
+    padding-right: 24px;
+    padding-bottom: 60px;
+  }
+
+  .sub-title-2 {
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 130%;
+    width: 100%;
+    justify-content: center;
+    padding-top: 100px;
+    padding-bottom: 24px;
+  }
+
+  .vector {
+    justify-content: center;
+    flex-grow: 2;
+  }
+
+  .img-recent-works {
+    padding-top: 80px;
+    justify-content: center;
+  }
+
+  .sub-title-3 {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 32px;
+    color: #091e42;
+    padding-top: 24px;
+  }
+
+  .infography-works {
+    font-family: 'Inter', sans-serif;
+    font-size: 16px;
+    color: #091e42;
+    padding-top: 12px;
+  }
+
+  .list-technologies-ul {
+    width: 100%;
+    flex-direction: row;
+    gap: 12px;
+    align-items: center;
+    padding-top: 12px;
+  }
+
+  .list-technologies-ul li {
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    padding: 10px 12px;
+    border-style: solid;
+    border: 1px solid #8993a4;
+    color: #091e42;
+  }
+
+  .button-cont {
+    padding-top: 12px;
+  }
+
+  .button-project {
+    font-family: 'Inter', sans-serif;
+    letter-spacing: 0.03em;
+    color: #e5e5e5;
+    font-size: 15px;
+    font-weight: 700;
+    text-align: center;
+    background: #ff6b00;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+    padding: 12px;
+    width: 124px;
+    height: 48px;
+    border: none;
+  }
+
+  /* Cards section */
+
+  .cards {
+    background-color: #e5e5e5;
+    display: grid;
+    text-shadow: 1px 1px 2px ccc;
+    padding-left: 24px;
+    padding-bottom: 24px;
+  }
```

Comment:
- [ ] As per the previous reviewer's suggestion you need to add some margin to the right side of the cards to match the given design. You can add `margin-right: ` here to see the effect and adjust it.
- ![image](https://github.com/Rebelzob/Portfolio/assets/60083437/afeb147a-2be6-4549-8740-2508b3df4bae)

Reply:
thanks for the review, I misunderstood that particular change the previous reviewer was pointing out.